docs/Tutorials/demo/budget_with_farming.md

```diff
@@ -195,11 +195,11 @@ farmingd q bank balances cosmos1228ryjucdpdv3t87rxle0ew76a56ulvnfst0hq0sscd3nafg
 
 ### Step 5. Send a governance proposal to create a public ratio amount plan
 
-Let's create `public-ratio-plan-proposal.json` file and copy the below JSON into a file.
+Now, create the `public-ratio-plan-proposal.json` file and copy the following JSON contents into the file. 
```

Comment:
does the location of the file matter? if yes, then specify the file location and folder name

Reply:
The location doesn't matter.